mozilla/kuma
# Fixes #5892, create header css bundle

@peterbe This separates out styling with regards to the header on the new frontend. We then use this new bundle separately inside `jinja2/react_base.html` and `kuma/wiki/jinja2/wiki/react_document.html`

Best way to test is:

1. Delete the `/static/` folder
2. `docker-compose exec web make build-static`
3. `docker-compose restart`
4. Visit the landing page, wiki pages, and search results page

Ensure that the header renders as expected in all cases. r?
